app/views/idv/cac/choose_method.html.erb

```diff
@@ -1,23 +1,24 @@
-<h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
-<%= t('cac_proofing.info.choose_method') %>
-<hr/>
-<br/>
-<%= image_tag asset_url('dod-logo.jpg'), height: 150 %>
-<br/>
-<%= simple_form_for :cac,
-                    url: url_for,
-                    method: 'put',
-                    html: { autocomplete: 'off', role: 'form', class: 'mt2' } do |f| %>
-  <button class="btn btn-primary" type="submit"><%= t('cac_proofing.buttons.use_cac') %></button>
+<%= render 'shared/maintenance_window_alert' do %>
+  <h1 class="h3 mt0 mb1"><%= t('cac_proofing.headings.choose_method') %></h1>
```

Comment:
this is just indenting & wrapping in a block (recommend that you hide whitespace for this diff)